tests/src/test/java/alluxio/rest/TestCase.java

```diff
@@ -102,6 +102,13 @@ public URL createURL() throws Exception {
             + "?" + sb.toString());
   }
 
+  /**
+   * Get the String from the inputStream of HttpURLConnection by BufferedReader
```

Comment:
I think you can remove this line (and the blank line below this), since the `@return` line already states what is returned. (The checkstyle is failing because this line does not end with a period).